node/src/test/embedding/openai.ts

```diff
@@ -47,4 +48,10 @@ describe('OpenAPIEmbeddings', function () {
       assert.deepEqual(vectors[1], stubValue.data.data[1].embedding)
     })
   })
+
+  describe('isEmbeddingFunction', function () {
+    it('should match the isEmbeddingFunction guard', function () {
+      assert.isTrue(isEmbeddingFunction(new OpenAIEmbeddingFunction('text', 'sk-key')))
```

Comment:
How does this test the fix? is it still two attributes?

Reply:
Is there a `isinstance()` type of call here? Can we rely on type system instead of check two arbitrary fields from the interface / type ?